### ISSUE TITLE
Payment Mode Deletion

### ISSUE THREAD

**Issue by inesmrad** [idurar/idurar-erp-crm#1021]
**Describe the bug**
After creating a payment mode and choosing to delete it , i get a confirmation pop up .Once i click on ok i get an error message "You can't delete payment mode after it has been created" .If i can't delete it , why would we get a confirmation message from the start ?

**To Reproduce**

**Expected behavior**
If the payment mode is actually never meant to be deleted, we shouldn't have the bin icon in the table .
If the payment mode is meant to be deleted, it should be deleted without error messages

**Screenshots**
![Screenshot (609)](https://github.com/idurar/idurar-erp-crm/assets/133030569/2f2f5207-4a9d-40e6-9a27-c2c09d8e045c)
![Screenshot (608)](https://github.com/idurar/idurar-erp-crm/assets/133030569/3cb817c7-da21-4846-b351-0a2c859473da)

**Desktop (please complete the following information):**
 - OS: [windows 11]
 - Browser [ chrome]
 - Version [123.0.6312.59]

**Comment by Vishrut19** [comment]
I can fix it easily. Can I take up this issue ?

**Comment by inesmrad** [comment]
> I can fix it easily. Can I take up this issue ? @inesmrad

yeah sure but we should get a confirmation as was the payment mode actuallly meant to be deleted or not

**Comment by Ouchi-Ranim** [comment]
This request throws a 403 error it means the server understands the request that the browser is making, but the server will not fulfill it since we lack the permission to delete it
